crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -132,6 +132,7 @@
 )]
 #![cfg_attr(not(test), warn(unused_crate_dependencies))]
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+#![feature(stmt_expr_attributes)]
```

Comment:
```suggestion
```

Reply:
done in 0afc55670748e75ab55169e30b491ceac640bac8

Reply:
Tests are failing because we are missing this `nightly` feature